### PR TITLE
Heroku config

### PR DESCRIPTION
**Pull request by ttttsai**
Thanks

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DB_URL = mongodb://localhost
+DB_PORT = 12345
+DB_NAME = anything
+DB_COLLECTION = anything
```
Comment (beltekylevi) [inline .env.sample]: why do you specify the collection here?

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,4 @@
+DB_URL = mongodb://localhost
```
Comment (beltekylevi) [inline .env.sample:1]: 🍬 you can separate the protocol from the address as well

(i'll use the 🍬_(candy emoji)_ if something is just an opinion)

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,6 @@
 .idea
 node_modules/
 dist/app
+.env
 *.log
 .application-config.json
```
Comment (beltekylevi) [inline .gitignore]: is this still necessary to ignore?

Comment (ttttsai) [reply .gitignore]: Yes, the documentation strongly recommend not to commit .env file.

Comment (beltekylevi) [reply .gitignore]: good answer, but it was not a tricky question 😄  i was thinking about the `.application-config.json`

**Review by beltekylevi — APPROVED** [review]

**Pull request merged**